src/test/kotlin/CheckersModelTest.kt

```diff
@@ -0,0 +1,40 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertTrue
+
+class TestCanMove {
+
+    @Test
+    fun `Test Move of Ordinary Checker 1`() {
+        val m = CheckersModel()
+        assertTrue(m.canMove(BaseTurn(1, Pair(0, 0), Pair(1, 1))) == null,
+            "Moved into not empty square")
+        assertTrue(m.canMove(BaseTurn(-1, Pair(5, 0), Pair(4, 1))) == null,
+            "Wrong player made turn")
+        assertTrue(m.canMove(BaseTurn(1, Pair(2, 2), Pair(3, 3))) === m.board[2, 2],
+            "Wrong square returned: start excepted")
+
+        m.move(BaseTurn(1, Pair(2, 2), Pair(3, 3)))
```

Comment:
Check the board state after you've moved, make the test more complete)

Reply:
Fixed